src/Form/InputClassifiersTypes.d.ts

```diff
@@ -1,14 +1,14 @@
 interface Matcher {
     type: string,
     selector: string,
-    regex: RegExp,
-    negativeRegex?: RegExp
+    matcherFn: (string) => boolean
```

Comment:
Instead of these regexes, I'm now using matcher functions so we have more flexibility with how we run these checks.